src/plugins/charts/public/services/theme/theme.test.tsx

```diff
@@ -125,7 +125,7 @@ describe('ThemeService', () => {
         setUpMockTheme.theme$ = createTheme$Mock(true);
         themeService.init(setUpMockTheme);
       });
-      expect(renderCounter).toHaveBeenCalledTimes(2);
+      expect(renderCounter).toHaveBeenCalledTimes(4);
```

Comment:
why is not called/renderer 4 times instead of 2?

Reply:
I was attempting to not directly mutate the `LIGHT_THEME` and `DARK_THEME` objects for the background overrides, but it's was a deep assignment anyways to I just removed it. Fixed in 292f3dec368d9d9e3ca3b8212a2f2f8fc5d6de98